### DISCUSSION TITLE
the Root Directory must be a relative path

### DISCUSSION THREAD

**Discussion by hassaanp** [vercel/community#1263] [Help] [answered]
Hi!

I am having an issue when trying to deploy a NextJS 13 application via Vercel. When I click the deploy button, I receive the following error message:

```
If defined, the Root Directory must be a relative path not starting with `./` and not including `../` or other special characters.
```

The root directory is automatically set as `.`, which is the default setting. I am not sure if this is a bug or if I am doing something wrong. If anyone has any ideas or suggestions on how to fix this issue, I would greatly appreciate it.

Thank you!

**Comment by amyegan** [comment, marked as answer]
Sorry for the delay @hassaanp! Is this a new project, or an existing project you're upgrading to version 13? By default, the Root Directory field should be empty for recently created projects.

<img src="https://user-images.githubusercontent.com/8247105/210613003-82058b8e-d34f-4ccb-a6f6-699812aaf3c0.png" alt="Root Directory setting showing field should be empty in code is not in a subdirectory" />

If you clear the Root Directory field, does the deployment succeed?

**Reply by hassaanp** [reply]
Hi!

This is a new Next 13 project.
I was using the Nextjs template, which was not allowing me to leave the root blank. I then decided to use the vercel cli to deploy successfully.

I believe choosing the `Other` option would have also worked.

**Reply by amyegan** [reply]
Can you share a link to the template you're using? I want to make sure we get it updated if the configuration is not correct 🙏
